src/screens/OrgList/OrganizationModal.tsx

```
@@ -271,15 +273,23 @@ const OrganizationModal: React.FC<InterfaceOrganizationModalProps> = ({
             name="photo"
             type="file"
             multiple={false}
-            onChange={async (e: React.ChangeEvent): Promise<void> => {
+            onChange={async (e: React.ChangeEvent) => {
               const target = e.target as HTMLInputElement;
               const file = target.files && target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)